docs/yaml/elementary/str.yml

```diff
@@ -312,6 +314,13 @@ methods:
     '3.6'.version_compare('>=3', '<4.0') == true
     ```
 
+    *Since 1.8.1* multiple versions can be compared using comma-separated string:
```

Comment:
This duplicates the section immediately above, which says that we now support multiple versions as multiple arguments. This was added in #14105 and aligns with the way e.g. `dependency(..., version: [first, second])` supports checking against multiple versions.

Reply:
> dependency(..., version: [first, second]) supports checking against multiple versions.

Yep. It works fine for dependency, but not for meson :( Making meson as dependency looks strange, otherwise I do not think how to do it with the current master branch. 

Other way is to implement array support for meson_version but it will require some changes which might be everywhere since 
https://github.com/mesonbuild/meson/blob/f36d680b0093e071ca54c9d20b84f9a62f88b94b/mesonbuild/interpreter/interpreter.py#L505-L508
accepts only strings (`pv` argument).